REVIEW_PROCESS.md

```diff
@@ -0,0 +1,21 @@
+# Review Process Guidelines
+
+Refer to https://help.github.com/articles/about-pull-request-reviews/ for details on Pull Request review process at GitHub.
+
+For Host OS project, also consider the following guidelines:
+
+## Avoid stacked Pull Requests
+
+* Pull Requests (PR) should be independent of each other.
+* If any, identify dependencies in the very first line of the PR description.
+* GitHub does not allow approving only a sub-set of commits from a PR. When commits are updated but the reviewing commit is not, there is no way to distinguish this in the GitHub interface.
```

Comment:
- sub-set => subset
- This is the motivation of the first guideline, I suggest creating a sub item in it instead of creating a new item as you are doing